datasets/modeled/buildings/dataset.json5

```diff
@@ -0,0 +1,116 @@
+{
+  dataset_id: "decarb_2023_buildings",
+  dataset_type: "modeled",
+  sector_description: "",
+  description: "",
+  origin_creator: "Aven Satre Meloy",
+  origin_organization: "",
+  origin_contributors: [
+  ],
+  origin_project: "",
+  origin_date: "",
+  origin_version: "",
+  data_source: "",
+  source: "",
+  data_classification: "low", // TODO: confirm
+  tags: [
+  ],
+  trivial_dimensions: [
+    "scenario",
+    "subsector",
+    "weather_year",
+  ],
+  data_schema: {
+    table_format: {
+      format_type: "unpivoted",
+      value_column: "value",
+    },
+    data_schema_type: "one_table",
+  },
+
+  dimensions: [
+    {
+      type: "geography",
+      name: "Region",
+      display_name: "Region",
+      file: "dimensions/regions.csv",
+      module: "dsgrid.dimension.standard",
+      class: "GeographyDimensionBaseModel",
+      description: "",
+    },
+    {
+      type: "metric",
+      name: "Building End Uses",
+      display_name: "Building End Uses",
+      file: "dimensions/enduses.csv",
+      module: "dsgrid.dimension.standard",
+      class: "EnergyEndUse",
+      description: "",
+    },
+    {
+      type: "model_year",
+      name: "Model Year 2018",
```

Comment:
This doesn't look right--there are three different model years in model_years.csv

Reply:
I think what @daniel-thom wast trying to convey here is that the model year is 2018 because all of the hourly results are based on ResStock and ComStock load profiles simulated for the year 2018. But I'm not sure whether this field is supposed to note the projection years, as well (and/or how to separately convey that those pertain to stock turnover/adoption but still use hourly profiles from a 2018 model year simulated data set).